### DISCUSSION TITLE
Welcome to apex-ag-grid-plugin Discussions!

### DISCUSSION THREAD

**Discussion by phartenfeller** [phartenfeller/apex-ag-grid-plugin#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aysearasghb** [comment]
Hello,
I can't install the plugin. When I try to upload a file it says "The file does not have any content." I get an error. How can I progress?

**Reply by phartenfeller** [reply]
Hi. Which file did you upload?

**Reply by aysearasghb** [reply]
I got it, there was a little problem uploading the file. Thanks for your return.
